src/Hl7.Fhir.Base/ElementModel/Types/Date.cs

```diff
@@ -141,35 +141,48 @@ private static bool tryParse(string representation, out Date value)
             return success;
         }
 
+        public static Date operator -(Date dateValue, Quantity addValue)
```

Comment:
I would suggest naming the quantity here as `subtractValue` instead of `addValue`.